### PR TITLE
docs: add ability to use headers to print query plans

### PR DESCRIPTION
**Pull request by smyrick**
Update docs on what headers can be set to return query plans in extensions

### CODE REVIEW

**Comment by smyrick** [comment]
@apollographql/docs Ping for review

File: docs/source/query-plans.mdx [hunk 1]
```diff
@@ -444,6 +444,14 @@ You can view the query plan for a particular operation in any of the following w
     * Note that you must [register your federated graph](./managed-federation/setup/) with GraphOS to view query plans in the Explorer.
 * As direct output from the `@apollo/gateway` library (see below)
 
+### Outputting query plans with headers
+In the latest versions of Apollo Router and Gateway you can pass the following headers to return the query plans in the response extensions:
```
Comment (Meschreiber) [inline docs/source/query-plans.mdx:448]: Could you please specify the versions to keep this more evergreen?

File: docs/source/query-plans.mdx [hunk 2]
```diff
@@ -444,6 +444,14 @@ You can view the query plan for a particular operation in any of the following w
...
+
+* Sending `Apollo-Query-Plan-Experimental` will return the query plan in the response extensions
```
Comment (Meschreiber) [inline docs/source/query-plans.mdx:450]:
```suggestion
* Including the `Apollo-Query-Plan-Experimental` header returns the query plan in the response extensions
```

File: docs/source/query-plans.mdx [hunk 3]
```diff
@@ -444,6 +444,14 @@ You can view the query plan for a particular operation in any of the following w
...
+* Sending `Apollo-Query-Plan-Experimental-Format` with one of the supported options will change the format:
```
Comment (Meschreiber) [inline docs/source/query-plans.mdx:451]: Is this one or the other, or most users pass both headers to select a format? Could you please add a <Note> below to clarify?

Comment (Meschreiber) [reply docs/source/query-plans.mdx:451]:
```suggestion
* Sending `Apollo-Query-Plan-Experimental-Format` with one of the supported options changes the output format:
```

File: docs/source/query-plans.mdx [hunk 4]
```diff
@@ -444,6 +444,14 @@ You can view the query plan for a particular operation in any of the following w
...
+  * A value of `prettified` will return the human readable formatted string
```
Comment (Meschreiber) [inline docs/source/query-plans.mdx:452]:
```suggestion
  * A value of `prettified` returns a human-readable string  of the query plan
```

File: docs/source/query-plans.mdx [hunk 5]
```diff
@@ -444,6 +444,14 @@ You can view the query plan for a particular operation in any of the following w
     * Note that you must [register your federated graph](./managed-federation/setup/) with GraphOS to view query plans in the Explorer.
 * As direct output from the `@apollo/gateway` library (see below)
 
+### Outputting query plans with headers
+In the latest versions of Apollo Router and Gateway you can pass the following headers to return the query plans in the response extensions:
+
+* Sending `Apollo-Query-Plan-Experimental` will return the query plan in the response extensions
+* Sending `Apollo-Query-Plan-Experimental-Format` with one of the supported options will change the format:
+  * A value of `prettified` will return the human readable formatted string  
+  * A value of `internal` will return the JSON representation
```
Comment (Meschreiber) [inline docs/source/query-plans.mdx:453]:
```suggestion
  * A value of `internal` returns a JSON representation of the query plan
```

**Review by Meschreiber — CHANGES_REQUESTED** [review]
Beyond the request to specify the versions explicitly, left some suggestions to use present rather than future tense.

File: docs/source/query-plans.mdx (hunk 1 above)
Comment (smyrick) [reply docs/source/query-plans.mdx]: Updated!

File: docs/source/query-plans.mdx [hunk 6]
```diff
@@ -444,6 +444,14 @@ You can view the query plan for a particular operation in any of the following w
     * Note that you must [register your federated graph](./managed-federation/setup/) with GraphOS to view query plans in the Explorer.
 * As direct output from the `@apollo/gateway` library (see below)
 
+### Outputting query plans with headers
+In the Apollo Router v0.16.0+ and Apollo Gateway v2.5.4+, you can pass the following headers to return the query plans in the GraphQL response extensions:
```
Comment (Meschreiber) [inline docs/source/query-plans.mdx:448]:
```suggestion
### Outputting query plans with headers

With [the Apollo Router](/router/) `v0.16.0+` and [`@apollo/gateway`](/apollo-server/using-federation/api/apollo-gateway/) `v2.5.4+`, you can pass the following headers to return the query plans in the GraphQL response extensions:
```

**Review by Meschreiber — APPROVED** [review]

File: docs/source/query-plans.mdx [hunk 7]
```diff
@@ -444,6 +444,15 @@ You can view the query plan for a particular operation in any of the following w
     * Note that you must [register your federated graph](./managed-federation/setup/) with GraphOS to view query plans in the Explorer.
 * As direct output from the `@apollo/gateway` library (see below)
 
+### Outputting query plans with headers
+
+With [the Apollo Router](/router/) `v0.16.0+` and [`@apollo/gateway`](/apollo-server/using-federation/api/apollo-gateway/) `v2.5.4+`, you can pass the following headers to return the query plans in the GraphQL response extensions:
+
+* Including the `Apollo-Query-Plan-Experimental` header returns the query plan in the response extensions
+* Additonally, including `Apollo-Query-Plan-Experimental-Format` with one of the supported options changes the output format:
```
Comment (Meschreiber) [inline docs/source/query-plans.mdx:452]:
```suggestion
* Additionally including the `Apollo-Query-Plan-Experimental-Format` header with one of the supported options changes the output format:
```

**Pull request merged**